app/build.gradle.kts

```diff
@@ -153,6 +154,9 @@ dependencies {
     androidTestImplementation("androidx.compose.ui:ui-test-junit4")
     debugImplementation("androidx.compose.ui:ui-tooling")
     debugImplementation("androidx.compose.ui:ui-test-manifest")
+
```

Comment:
If possible, put implementation dependencies next to other implementations and not together with test/debug dependencies.

Also, accompanist-permissions is made for permissions 'cleaner' handling, if I am not mistaken. It is not really specific to notifications, could put it just under android group somewhere above.